src/utils/etherspot.js

```diff
@@ -284,3 +285,37 @@ export const getChainTokenListName = (chain: Chain): ?string => {
 
   return null;
 };
+
+/**
+ * How use this function
+ * @param {*} assets is mendatory. like own tokens, default tokens
+ * @param {*} chain is optional. using chain to find perticular chain assets
+ * @param {*} stableTokens is optional. check assets in this tokens availability
+ * @param {*} filteringAssets is optional. mostly default token list or default stable list.
+ * @returns if assets is null then return blank array. otherwise return filter tokens list
+ */
+export const filteredChainAssets = (
+  assets: AssetsPerChain,
+  chain?: Chain,
+  stableTokens?: Asset[],
+  filteringAssets?: Asset[],
+) => {
+  if (!assets || isEmpty(assets)) return [];
+
+  const newAssets = chain ? assets.filter((asset) => asset.chain === chain) : assets;
+
+  if (stableTokens) {
+    return assets.filter((assetToken) => stableTokens?.some((stableToken) => isSame(assetToken, stableToken)));
+  }
+
+  if (filteringAssets) {
+    const filterDefaultAssets: any = filteringAssets.filter(
```

Comment:
Try avoiding `any` where you can. In this case you know that `filteringAssets` has the type `Assets[]`, so that's what will be returned from `.filter`